src/containers/Confirmation/ConfirmationProvider.tsx

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+import type { ReactNode } from 'react'
+import { useState } from 'react'
+
+import type { Confirm } from './context'
+import { ConfirmationContext } from './context'
+import ModalDialog from '../../components/ModalDialog'
+
+export type Props = {
+  children: ReactNode
+}
+export const ConfirmationProvider = ({ children }: Props) => {
+  const [confirm, setConfirm] = useState<Confirm>({
+    prompt: '',
+    title: '',
+    isOpen: false,
+    proceed: () => {},
+    cancel: () => {},
+  })
+
+  return (
+    <ConfirmationContext.Provider value={{ confirm, setConfirm }}>
+      {children}
+      <ModalDialog
+        open={confirm.isOpen}
```

Comment:
we currently have open and isOpen, would just choose one (second my fav)

Reply:
Can't change 'open', that's a prop coming from an ASC-component. Changed isOpen to open.